main.go

```diff
@@ -117,21 +117,24 @@ func init() {
 		RunE:    contextualize(handler.Variables, handler.Panic),
 	})
 	variablesCmd.AddCommand(&cobra.Command{
-		Use:   "get",
-		Short: "Get the value of a variable",
-		RunE:  contextualize(handler.VariablesGet, handler.Panic),
-		Args:  cobra.MinimumNArgs(1),
+		Use:     "get key",
```

Comment:
What the heck? I always thought this was just the command name to use for Cobra. I didn't know you could put help strings afterwards

Reply:
Ya me neither lol. Guess we know now!

![image](https://user-images.githubusercontent.com/5499880/119913610-35090700-bf13-11eb-9532-058a206f27bb.png)